packages/generator-volto/.eslintrc.js

```diff
@@ -1,9 +1,9 @@
 module.exports = {
   // extends: ['eslint:recommended', 'prettier'],
-  plugins: ['prettier'],
-  rules: {
-    'prettier/prettier': 'error',
-  },
+  // plugins: ['prettier'],
+  // rules: {
+  //   'prettier/prettier': 'error',
+  // },
```

Comment:
Do not comment out tools that are required for CI. Please revert.

Reply:
Dont worry, This was just for testing purposes.